doc/classes/ProjectSettings.xml

```diff
@@ -1756,6 +1756,10 @@
 		<member name="rendering/limits/spatial_indexer/update_iterations_per_frame" type="int" setter="" getter="" default="10">
 		</member>
 		<member name="rendering/limits/time/time_rollover_secs" type="float" setter="" getter="" default="3600">
+			When the [code]TIME[/code] shader built-in variable is incremented, it will roll over to [code]0.0[/code] when the value specified in [member rendering/limits/time/time_rollover_secs] is exceeded (in seconds). Since large floating-point values are less precise than small floating-point values, this should be set as low as possible to maximize the precision of the [code]TIME[/code] built-in variable in shaders. However, if this is set too low, shader animations may appear to restart from the beginning while the project is running.
+		</member>
+		<member name="rendering/limits/time/time_rollover_secs.mobile" type="int" setter="" getter="" default="30">
```

Comment:
The property hint doesn't seem to apply to the override, it's typed as `int` (and likely doesn't have a range slider).

Also, why 30 seconds specifically? That sounds really low.

Reply:
> Also, why 30 seconds specifically? That sounds really low.

@clayjohn recommended to use 30 instead of 60, as 16-bit floating point precision gets low *very* quickly. If the rollover time was 60, the worst precision you'd get would be `0.03`, and this is already too imprecise for many shader effects: https://en.wikipedia.org/wiki/Half-precision_floating-point_format#Precision_limitations